aten/src/ATen/native/Fill.cpp

```diff
@@ -16,7 +16,7 @@ Tensor& fill_out(Tensor& self, const Scalar value) {
   return self;
 }
 
-Tensor& fill_(Tensor& self, Scalar value) {
+Tensor& fill_(Tensor& self, const Scalar value) {
```

Comment:
We don't usually label value types const

Reply:
(but we're not very consistent about it)

Reply:
I changed this to consistently not use const